lib/parse-args.js

```diff
@@ -145,6 +151,18 @@ function buildYargs (withCommands = false) {
     })
     .epilog('visit https://git.io/vHysA for list of available reporters')
 
+  yargs.middleware((argv) => {
+    if (!argv[100]) return argv
+
+    return {
+      ...argv,
+      branches: 100,
+      functions: 100,
+      lines: 100,
+      statements: 100,
+    }
+  })
```

Comment:
this code doesn't seem to be executed. I haven't worked with yargs middleware before but it seems like a good approach here? Do you know why the code is not executed?

Reply:
This seems like a reasonable approach to me, and seems like it should work, perhaps:

```

yargs.middleware((argv) => {
    if (!argv['100']) return argv

    return {
      ...argv,
      branches: 100,
      functions: 100,
      lines: 100,
      statements: 100,
    }
}
```

I wouldn't expect the argument key itself to be coerced into an integer.

Reply:
my problem here is that the middleware is not applied at all. I think I'm missing something obvious. Any idea what it could be?

Reply:
I tested locally, it seems like this is a bug that I fixed in yargs@17, at least it works as expected with yargs@17 ...

Unfortunately, I think we might want to hold off on dropping Node 10 until I can drop Node 10 on the libraries I manage at Google.

We can implement a less elegant version here:

https://github.com/bcoe/c8/blob/main/lib/commands/report.js#L29

Which I believe should work in the appropriate contexts (we'll want to confirm it works for the `check-coverage` command).

Let's add a `// TODO: comment` to switch to the `middleware` approach as soon as we can drop Node 10?